libs/ui/src/unlock_machine_screen.tsx

```diff
@@ -23,16 +22,19 @@ const NumberPadWrapper = styled.div`
   }
 `;
 
-export const EnteredCode = styled.div`
+const EnteredCode = styled.div`
   margin-top: 5px;
   text-align: center;
   font-family: monospace;
   font-size: 1.5em;
   font-weight: 600;
 `;
 
-export function UnlockMachineScreen(): JSX.Element {
-  const { auth, electionDefinition, machineConfig } = useContext(AppContext);
+interface Props {
+  auth: DippedSmartcardAuth.CheckingPasscode;
```

Comment:
I believe we can also expect to receive an `InsertedSmartcardAuth.CheckingPasscode` auth object here. For example, on BMDs, admins will still use inserted smartcard auth instead of dipped smartcard auth. Likewise, poll worker cards will soon be able to have PINs (for VVSG2 cert).

(cc @jonahkagan)